std_msgs/Header header
uint32 uav_count
geometry_msgs/Pose[] uav_positions
